app/src/main/java/net/gsantner/opoc/util/CoolExperimentalStuff.java

```diff
@@ -141,4 +143,44 @@ public void run() {
             w.setLayout(WindowManager.LayoutParams.MATCH_PARENT, WindowManager.LayoutParams.MATCH_PARENT);
         }
     }
+
+    private static Thread autoScrollerThread = null;
+
+    public static void startAutoScroller(final Activity activity, final boolean isWebView) {
```

Comment:
Change to View, then check 

`instanceof EfitText` and `WebView`.

Pass from fragment hlEditor or webview (depending on current shown). 

This way, you dont need to lookup anything, and dont need the boolean argument.